elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -509,6 +521,7 @@ export class PeerPool extends EventEmitter {
 		// Clear periodic eviction of outbound peers for shuffling
 		if (this._outboundShuffleIntervalId) {
 			clearInterval(this._outboundShuffleIntervalId);
+			clearInterval(this._outboundUpdateStatusId);
```

Comment:
What is the reason to update the status ID at this point?

Reply:
statusId is created in the constructor for setInterval so that we can clear it when we are stopping the p2p node on cleanup